examples/ReadWriteSplittingSample/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
do you need a new folder or a new package inside AWSDriverExample ?

Reply:
Sorry, I'm not sure what you are suggesting, can you clarify? I believe the ReadWriteSplittingSample directory matches our other example directory structures, eg HikariExample. Are you suggesting I move this to the AWSDriverExample directory instead of having it in its own directory? My concerns with doing that are that AWSDriverExample contains single files and the logging config in ReadWriteSplittingExample is pretty specific to just ReadWriteSplittingSample.java